.gitattributes

```diff
@@ -1,8 +1,8 @@
 # Auto detect text files and perform LF normalization
-* text=auto
+* text=auto eol=crlf
```

Comment:
this is required to get Linux to honour the source code's crlf encoding regardless of what Linux would like to do